### PR TITLE
## In this pull request, I have implemented features mentioned in the requirement for the accessibility of project.

### PR DESCRIPTION
**Pull request by sanamrahimi**
#Page titles - no issues found
#image text alternative - no issues found
#Text heading - no issues found
#color contrast - no issues found
#Resize - no issues found
#Interaction - add a label to contact form [link](https://github.com/sanamrahimi/protfolio/pull/7/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L194-R195)
#Moving content - no issues found
#Multimedia - no issues found
#The basic structure of the page-no issues found

### CODE REVIEW

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @sanamrahimi, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

You have submitted a pair programming project without the commits of other team member. Kindly ensure that every member of the team participates in the project evidenced by their commits.

Also, the pull request seems not to contain any changes relating to the milestone.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by huseyinbiyik** [comment]
Hi team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

⚠️  Please remove the instructional texts from your README. They are helper texts for you to set up a proper README. 
Screenshot 2023-04-11 at 10 35 34 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <title>Sanam's Protfolio</title>
```
Comment (huseyinbiyik) [inline index.html:8]: According the importance of element, I think the image I added below describe the headings hierarchy better. You used h2 element for the different elements with different importance and sizes. Depending on the Figma font-size, that would be a better order. Kindly fix them.
Screenshot 2023-04-11 at 23-20-18 Sanam's Protfolio

File: index.html [hunk 2]
```diff
@@ -191,9 +192,9 @@ <h2>Skills</h2>
     <section class="contact-me">
         <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
     <form action="https://formspree.io/f/mnqyglkn" method="post">
-        <input type="text"  id="name" name="user_name" placeholder="Full name" maxlength="30" required>
-        <input type="email"  id="email" name="use_email" placeholder="Email adress" required>
-        <textarea type="text" id="message" name="message" cols="30" rows="5" placeholder="Hey I'd like to connect to" maxlength="500" required></textarea>
+        <label for="name"><input type="text"  id="name" name="user_name" placeholder="Full name" maxlength="30" required></label>
+        <label for="email"><input type="email"  id="email" name="use_email" placeholder="Email adress" required></label>
+        <label for="message"><textarea type="text" id="message" name="message" cols="30" rows="5" placeholder="Hey I'd like to connect to" maxlength="500" required></textarea></label>
         <button type="submit">Get in Touch</button>
```
Comment (huseyinbiyik) [inline index.html:198]: Kindly make sure there is no error in the contrast check. There is a small one with the buttons, kindly add different colors for the button states to increase accessibility. Here is the [result](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2023-04-11-at-20-07-54-499?failing) of the error. Don't forget to add the change to the related section to your pull request description.
Screenshot 2023-04-11 at 11 33 12 PM

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
Almost there

## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (BenMKT) [reply index.html:8]: -Rectified `headings` hierarchy with respect to the importance  of the headings per section

File: index.html (hunk 2 above)
Comment (BenMKT) [reply index.html:198]: -Added `background-color` to buttons on different states specifically when pressed to increase accessibility solving above raised bug.

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @sanamrahimi 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**